api/Window.json

```
@@ -4123,15 +4123,33 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Window/navigator",
           "spec_url": "https://html.spec.whatwg.org/multipage/system-state.html#dom-navigator",
           "support": {
-            "chrome": {
-              "version_added": "1"
-            },
-            "chrome_android": {
-              "version_added": "18"
-            },
-            "edge": {
-              "version_added": "12"
-            },
+            "chrome": [
+              {
+                "version_added": "1"
+              },
+              {
+                "version_added": true,
+                "alternative_name": "clientInformation"
+              }
+            ],
+            "chrome_android": [
+              {
+                "version_added": "18"
+              },
+              {
+                "version_added": true,
```

```suggestion
                "version_added": "18",
```

Based on https://trac.webkit.org/changeset/1024/webkit (almost the beginning of WebKit)